Client/src/redux/reducers/titleReducer/titleReducer.js

```diff
@@ -0,0 +1,16 @@
+import { CHANGE_APP_TITLE } from "../../actions/types";
+
+const INITIAL_STATE = {
+    title: 'Weather App',
+};
+
+const titleReducer = (state = INITIAL_STATE, action) => {
+    switch (action.type) {
+        case CHANGE_APP_TITLE: return { ...state, title: 'Meteorology App' }
```

Comment:
do not store static data in redux. If title is never gonna change - remove it from here. Though I realize this was done for the sake of training.